<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control_Form</class>
 <widget class="QWidget" name="Control_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_online_control">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>在线控制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_offline_control">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>离线控制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_control_help">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_control_back_nagivator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回导航界面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
